exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/DataTypeProvider.kt

```diff
@@ -148,7 +148,7 @@ abstract class DataTypeProvider {
         }
 
         e is LiteralOp<*> -> (e.columnType as IColumnType<Any?>).valueAsDefaultString(e.value)
-        e is Function<*> -> "$e"
+        e is Function<*> && currentDialect !is SQLiteDialect -> "$e"
```

Comment:
In SQLite, complex functions need to be wrapped in parentheses. Since there is no way to know if an expression is a complex function or not, I decided to wrap all default expressions in parentheses for SQLite.